include/sndfile.h.in

```diff
@@ -220,8 +220,11 @@ enum
 
 	SFC_SET_VBR_ENCODING_QUALITY	= 0x1300,
 	SFC_SET_COMPRESSION_LEVEL		= 0x1301,
+
+	/* Ogg format commands */
 	SFC_SET_OGG_PAGE_LATENCY_MS		= 0x1302,
 	SFC_SET_OGG_PAGE_LATENCY		= 0x1303,
+	SFC_GET_OGG_STREAM_SERIALNO		= 0x1304,
```

Comment:
Hi @arthurt , thanks for great work! But I'm a little confused by this place, do we have `SFC_GET_BITRATE_MODE` and `SFC_GET_OGG_STREAM_SERIALNO` with the same value `0x1304`?

Reply:
Yup, that was a mistake. SFC_GET/SET_BITRATE_MODE probably should use values further away from the SFC_GET/SET_OGG_* values, but they already are shipped.

Fixed, thanks!